python/lsst/ts/watcher/alarm.py

```diff
@@ -272,6 +303,11 @@ def reset(self):
     def set_severity(self, severity, reason):
         """Set the severity.
 
+        Call the callback function unless the alarm was nominal
+        and remains nominal.
+        Put the new severity on the severity queue (if it exists),
+        regardless of whether the alarm was nominal.
+
         Parameters
         ----------
         severity : `lsst.ts.idl.enums.Watcher.AlarmSeverity` or `int`
```

Comment:
You imported `AlarmSeverity`. Is the whole package needed here then or can do you with only the class name? Further down (line 455) you only use the class name.

Reply:
Good question! I did some experiments and here is what I found:
* I had not been linking to the ts_idl docs at all: I added an intersphinx_mapping for ts_idl to conf.py.
* Once I added fixed that, I found that the fully qualified name was required to get a useful link. The short name didn't work. I can see why that might be, but it was a bit disappointing.